releng/tools/rmonitor

```diff
@@ -0,0 +1,41 @@
+#!python
```

Comment:
this shebang does not seem to work with bash. I suggest to use something like `#!/usr/bin/env python`